From 0cceb759b4eb66e9614421d17d6e58bef87f4e43 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 21:18:17 -0400
Subject: [scsi] ipr: Add new CCIN definition for new hardware support

Message-id: <1409606300-27001-4-git-send-email-gduarte@redhat.com>
Patchwork-id: 90131
O-Subject: [RHEL7.1 PATCH BZ 1088561 3/6] ipr: Add new CCIN definition for new hardware support
Bugzilla: 1088561
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088561
Upstream Status: f94d9964626fd75b22878cc97ffebf287415f64e

commit f94d9964626fd75b22878cc97ffebf287415f64e
Author: Wendy Xiong <wenxiong@linux.vnet.ibm.com>
Date:   Tue Jan 21 12:16:40 2014 -0600

    [SCSI] ipr: Add new CCIN definition for new hardware support

    Add the appropriate definition and table entry for new hardware support.

    Signed-off-by: Wen Xiong <wenxiong@linux.vnet.ibm.com>
    Acked-by: Brian King <brking@linux.vnet.ibm.com>
    Signed-off-by: James Bottomley <JBottomley@Parallels.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/ipr.c b/drivers/scsi/ipr.c
index 7e1276a..dfaa47f 100644
--- a/drivers/scsi/ipr.c
+++ b/drivers/scsi/ipr.c
@@ -9998,6 +9998,8 @@ static struct pci_device_id ipr_pci_table[] = {
  { PCI_VENDOR_ID_IBM, PCI_DEVICE_ID_IBM_CROCODILE,
   PCI_VENDOR_ID_IBM, IPR_SUBS_DEV_ID_57D9, 0, 0, 0 },
  { PCI_VENDOR_ID_IBM, PCI_DEVICE_ID_IBM_CROCODILE,
+  PCI_VENDOR_ID_IBM, IPR_SUBS_DEV_ID_57DA, 0, 0, 0 },
+ { PCI_VENDOR_ID_IBM, PCI_DEVICE_ID_IBM_CROCODILE,
   PCI_VENDOR_ID_IBM, IPR_SUBS_DEV_ID_57EB, 0, 0, 0 },
  { PCI_VENDOR_ID_IBM, PCI_DEVICE_ID_IBM_CROCODILE,
   PCI_VENDOR_ID_IBM, IPR_SUBS_DEV_ID_57EC, 0, 0, 0 },
diff --git a/drivers/scsi/ipr.h b/drivers/scsi/ipr.h
index f66645c..c756ff0 100644
--- a/drivers/scsi/ipr.h
+++ b/drivers/scsi/ipr.h
@@ -101,6 +101,7 @@
 #define IPR_SUBS_DEV_ID_57D7    0x03FF
 #define IPR_SUBS_DEV_ID_57D8    0x03FE
 #define IPR_SUBS_DEV_ID_57D9    0x046D
+#define IPR_SUBS_DEV_ID_57DA    0x04CA
 #define IPR_SUBS_DEV_ID_57EB    0x0474
 #define IPR_SUBS_DEV_ID_57EC    0x0475
 #define IPR_SUBS_DEV_ID_57ED    0x0499
-- 
1.7.1